### PR TITLE
Update _header.html.erb To make the language dropdown in login work

### PR DESCRIPTION
**Pull request by aayushgupta1**
The language dropdown in the login wasn't working. To make that work.

Make sure these boxes are checked before your pull request is ready to be reviewed and merged. Thanks!

* [x] all tests pass -- `rake test:all`
* [x] code is in uniquely-named feature branch, and has been rebased on top of latest master (especially if you've been asked to make additional changes)
* [x] pull request is descriptively named with #number reference back to original issue

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/wiki/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. Please alert developers on plots-dev@googlegroups.com when your request is ready or if you need assistance.

Thanks!

### CODE REVIEW

**Comment by aayushgupta1** [comment]
@jywarren  I was just going through the website. There was a bug where in the login tab, there's a language dropdown. But it's not working. So this should fix that. Please take a look at it.

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -181,7 +181,7 @@
     <li><a href="/people">Contributors</a></li>
 	           <li class="divider"></li>
 	       <% end %>
-              <li class="dropdown">
+              <ul class="dropdown">
```
Comment (jywarren) [inline app/views/layouts/_header.html.erb]: Hi, I think perhaps it has to be a `<ul>` inside of a `<li>` -- can you try that and also upload a screenshot of how it looks? You may want to test it out on a narrow window (to see how it'll look on a smartphone, too) as well.

Thank you very much!

**Comment by aayushgupta1** [comment]
@jywarren That's not the problem here. The problem is that when you click at the language dropdown, it vanishes. It doesn't pop down.

**Comment by jywarren** [comment]
Ah yes, I realize that, but we still need to respect correct html nesting
so can you try this too? Thanks!

**Comment by aayushgupta1** [comment]
Yeah. I get that. I was just giving this a try. I have reverted back to the original one( that you suggested ). I'll try and solve this issue. If you have any idea, do let me know.

**Comment by jywarren** [comment]
Oh, I'm not saying replace it, but try using both --`<li><ul></li></ul>` - make sense? I think it may preserve your fixand also respect nesting.

**Comment by aayushgupta1** [comment]
Yeah. Sure thing. I'll keep that in mind. Thanks!

**Comment by jywarren** [comment]
Hi, @aayushgupta1 - did you have a chance to try the nested version, to see if that works? Thanks!

**Comment by jywarren** [comment]
Hi, just checking in if you'd like me to take a crack at this? Thanks!

**Comment by aayushgupta1** [comment]
Sorry about that. Was just busy with my exams.

**Comment by jywarren** [comment]
No problem! Just wanted to make sure you weren't stuck or something. Thanks, and good luck on your exams!

**Comment by jywarren** [comment]
I looked into this and that change didn't seem to have the desired effect -- I'm not convinced it's possible to do a dropdown within a dropdown, at least with normal Bootstrap usage -- it's not in the docs. I'm going to close this out, esp. since there is a language menu in the footer. Thanks!

**Pull request closed without merging**